api/lang/server.go

```diff
@@ -180,6 +180,8 @@ func (h *LangHandler) Handle(ctx context.Context, conn jsonrpc2.JSONRPC2, req *j
 		return specs()
 	case "gauge/executionStatus":
 		return execution.ReadExecutionStatus()
+	case "gauge/getRunnerLanguage":
+		return getRunnerLanguage()
```

Comment:
follow the same convention as other requests. Can it be `gauge/runnerLanguage`  or some other name?